lib/utils/ConfigParser.js

```diff
@@ -272,6 +253,42 @@ class ConfigParser {
         return this._capabilities
     }
 
+    /**
+     * sets config attribute with file paths from filtering
+     * options from cli argument
+     *
+     * @param  {String} filterObj  list of files in a string from
+     * cli argument
+     * @return {Boolean} exclude  true or false
+     */
+    setFilePathToFilterOptions (filterObj, exclude = false) {
+        const FILTER_TYPE = exclude ? 'exclude' : 'specs'
+
+        if (typeof filterObj === 'string') {
+            const objs = new Set()
+            const objList = filterObj.split(/,/g)
+            const objsList = ConfigParser.getFilePaths(this._config[FILTER_TYPE])
+
+            for (let obj of objList) {
+                if (fs.existsSync(obj) && fs.lstatSync(obj).isFile()) {
+                    objs.add(path.resolve(process.cwd(), obj))
+                    continue
+                }
+                objsList.forEach(file => {
+                    if (file.match(obj)) {
+                        objs.add(file)
```

Comment:
in case of exclude=true shouldn't we not add the spec file here?

Reply:
we want to add it since that's how it will decide what files to exclude from the run

Reply:
right